libraries/service/src/main/java/org/holochain/androidserviceruntime/service/HolochainService.kt

```diff
@@ -497,6 +506,16 @@ class HolochainService : Service() {
                 )
                 startForeground()
             }
+            ACTION_BOOT_COMPLETED -> {
+                Log.d(
+                    logTag,
+                    "onStartCommand ACTION_BOOT_COMPLETED",
+                )
+
+                if (getAutostartConfigManager().isEnabled()) {
+                    startForeground()
```

Comment:
Upon receiving the Intent, the `HolochainService` checks that autostart is enabled, and then launches the service.